### ISSUE TITLE
Disabling terminal colours doesn't work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#56]
```
Disabling terminal colours as described on the know issues page doesn't
work. It gives multiple compiling errors with version 1.30b
```

Original issue reported on code.google.com by `ssky...@gmail.com` on 11 Apr 2010 at 8:01

**Comment by GoogleCodeExporter** [comment]
```
Whoops, true. Sorry about that. I have a fix and will include it in 1.31 in a 
couple 
of days. If you want a quick workaround, edit debug.h, and locate this line:

#  define cRST <end of line>

...and append space, then "" at the end:

#  define cRST ""

...this should help.

```

Original comment by `lcam...@gmail.com` on 12 Apr 2010 at 8:05
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thank you! :-) Workaround works.
```

Original comment by `ssky...@gmail.com` on 13 Apr 2010 at 12:58
